libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -510,13 +511,16 @@ private List<IdentityClaim> getIdentityClaimsForBirthDateCorrelation(
                     continue;
                 }
                 addLogMessage(item, "Birthdate property is missing from VC");
+                throw new HttpResponseExceptionWithErrorBody(
+                        HttpStatus.SC_INTERNAL_SERVER_ERROR,
+                        ErrorResponse.FAILED_BIRTHDATE_CORRELATION);
```

Comment:
I hadn't really considered, but worth noting that this means we treat missing fields differently to other correlation failures - in practice meaning that a missing birthdate will result in a technical error, but an incorrect birthdate will result in PYI_NO_MATCH.

That seems ok to me though - if a CRI starts omitting one of these fields, it's a bug that we want to notice!